install_config/install/deploy_router.adoc

```diff
@@ -635,6 +638,23 @@ $ oadm router \
 ====
 endif::[]
 
+
+=== F5 Router Partition Paths
+Partition paths allow you to store your OpenShift routing configuration in a
+custom *F5 BIG-IP®* administrative partition instead of the
+default */Common* partition.  Custom administrative partitions can be used
+to secure *F5 BIG-IP®* environments so that all OpenShift specific
+configuration stored in *F5 BIG-IP®* system objects resides within a
```

Comment:
Hmm, wouldn't it sound odd ```all OpenShift specific configurations``` in this context? Basically its referring to the configuration stored in the F5 system objects, so the plural form might sound odd, no?